scenarios/04_Battle_of_Barnon.cfg

```diff
@@ -417,42 +453,126 @@
                 [/if]
             [/then]
             [else]
+                # This is the case where no one has escaped yet; if player is still goofing off leaving Lorin & Reme on the battlefield,
+                # then they deserve this:
                 {VARIABLE_OP ano_tr add 1}
-                #	    {IF ano_tr equals 6} # To make it easier that Gawen cannot win
-#ifdef HARD
-                #		[gold]
-                #			side=3
-                #			amount=70
-                #		[/gold]
-#endif
-#ifdef NORMAL
-                #		[gold]
-                #			side=3
-                #			amount=60
-                #		[/gold]
-#endif
-#ifdef EASY
-                #		[gold]
-                #			side=3
-                #			amount=50
-                #		[/gold]
-#endif
-                #[gold]
-                #	side=4
-                #	amount=20
-                #[/gold]
-                #	    {END_IF_WITHOUT_ELSE}
+                {IF ano_tr equals 6} # To make it more certain that Gawen cannot win
+                [gold]
+                    side=3 # Bor Cryne
+                    amount={ON_DIFFICULTY 50 60 70}
+                [/gold]
+                [gold]
+                    side=4 # Uri Van Roe
+                    amount={ON_DIFFICULTY 10 20 30}
+                [/gold]
+                {END_IF_WITHOUT_ELSE}
             [/else]
         [/if]
+        # Keep the ON_DIFFICULTY parts of these synced with their initial income declarations above:
+        [modify_side]
+            side=2
+            income="$({ON_DIFFICULTY 7 9 11} + ($turn_number / 2))"
```

Comment:
Are you talking just side 2 here? I guess I could remove it for Hoyre since he's supposed to be easier to handle, but I definitely needed it for the other 2 enemy sides, as I found myself grinding to a stalemate without it. Even with the added enemy income I still managed to survive a good number of turns and get a decent bonus.

Reply:
No, I'm talking about all sides. Go ahead and keep this as is then, and I'll try and playtest it on my own to see how it plays out.